pkgs/racket-doc/scribblings/reference/generic.scrbl

```diff
@@ -146,15 +146,20 @@ Raised for @techlink{generic methods} that do not support the given
 
 @defform[(define/generic local-id method-id)]{
 
-When used inside the method definitions associated with the
-@racket[#:methods] keyword, binds @racket[local-id] to the generic for
-@racket[method-id]. This form is useful for method specializations to
-use generic methods (as opposed to the local specialization) on other
-values.
+When used inside the method definitions associated with the @racket[#:methods]
+, @racket[#:fallbacks], @racket[#:defaults] or @racket[#:fast-defaults] keyword,
```

Comment:
The comma should be on the preceding line. Otherwise, there will be a space between `@racket[#:methods]` and `,`.

Reply:
Thank you!